src/pages/api/salesConfirmation.ts

```diff
@@ -79,6 +81,15 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
             }
         } else if (fareType === 'flatFare') {
             userDataJson = getFlatFareTicketJson(req, res);
+        } else if (fareType === 'schoolService') {
```

Comment:
Could we move this check higher up, and if fareType === schoolService, we could change the fareType variable to be whatever it is inside the schoolFareType attribute, as then we could reduce the size of this if block ( it would call getSingleTicketJson or getMultipleServicesTicketJson or getFlatFareTicketJson as normal).